.github/workflows/node-zxc-compile-application-code.yaml

```diff
@@ -160,6 +160,7 @@ jobs:
   compile:
     name: ${{ inputs.custom-job-label || 'Compiles' }}
     runs-on: network-node-linux-large
+    if: ${{ !github.event.workflow_call.repository.fork }}
```

Comment:
This completely disables compilation checks on all forked PRs and these should run.

Reply:
Removed this if-check.